CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Changelog
+
+Useful links: Docsy [releases][] & [tags][]. Jump to the [latest][] release.
+
+## Upcoming release
+
+For a list of issues targeted for the next release, see the [22Q2][] milestone.
+
+## [0.2.0][]
+
+**New**:
+
+- Official Docsy support for [Hugo modules][]. Many thanks to the dedicated and
+  patient efforts of [@deining][], who researched, experimented, and implemented
+  the this feature. Thanks to [@deining][] and [@LisaFC][] for the
```

Comment:
and implemented the this feature -> and implemented this feature

Reply:
Fixed, thx!